package.json

```diff
@@ -32,6 +33,7 @@
     "jsonata": "^2.0.1",
     "loglevel": "^1.8.1",
     "minimist": "^1.2.7",
+    "mkdirp": "^2.1.5",
```

Comment:
I have a question.
This module is for using `## Working with Linked Enact` in README.md only(check line 51 in README.md).
And there's no use case in the jsdoc-to-ts module.
Should I add this module on package.json?

Reply:
I don't think we need to add this.